ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -314,6 +314,78 @@ private MainnetProtocolSpecs() {}
         .name("Istanbul");
   }
 
+  // TODO edwardmack, this is just a place holder definiton, REPLACE with real definition
+  public static ProtocolSpecBuilder<Void> atlantisDefinition(
```

Comment:
I would move these two methods into a new `ClassicProtocolSpecs` class. This class is focussed on providing the specs for Ethereum MainNet (we have separate classes to vary them as required for IBFT and Clique).

Admittedly the term `MainNet` is now somewhat overloaded with both Ethereum MainNet and ETC MainNet supported. There's always been a bit of overuse of the term MainNet anyway (e.g. `MainNetProtocolSpecs` are also the definitions for ropsten and `MainNetTransactionProcessor` is the only one we have etc). We've been considering doing a bunch of renames to solve that for a while so this may be another prompt to actually do it.

Reply:
Good idea, I'll create a new `ClassicProtocolSpecs` class.